### DISCUSSION TITLE
Accdb integration

### DISCUSSION THREAD

**Discussion by tristion** [Budibase/budibase#3464] [Ideas]
Some of us are stuck with more legacy databases that we'd like/have to access. :grinning:  By any chance, are there any plans to include an Accdb integration?
Thanks!

**Comment by mjashanks** [comment]
I had to google that! We are not going to be supporting it directly anytime soon - it's more likely that we'll open up the platform to allow you to write one yourself before we do it.

However, couldn't you import it into MS SQL Server (https://docs.microsoft.com/en-us/sql/integration-services/import-export-data/connect-to-an-access-data-source-sql-server-import-and-export-wizard?view=sql-server-ver15), and connect to that instead? MS SQL Server is supported by Budibase, and is about to get a great upgrade.

**Reply by tristion** [reply]
That would definitely be the way to go, and I would prefer it. Unfortunately our IT guy is the "if it ain't broke, don't fix/upgrade it" sort, so I'm stuck with what I've got. XD

That being said, I've found some utilities that will let me convert that database into something more friendly. It's not the _most_ ideal, since I'll have to do that once a day, but that's what automation is for so I'll probably go that route temporarily. 

Thanks for your reply!
